lib/skate/detours/route_segments.ex

```diff
@@ -4,6 +4,17 @@ defmodule Skate.Detours.RouteSegments do
   """
   alias Util.NearestPoint
 
+  defmodule UnfinishedResult do
+    @moduledoc false
+    @type t :: %__MODULE__{
+            before_start_point: [Util.Location.From.t()],
+            after_start_point: [Util.Location.From.t()]
+          }
+    @enforce_keys [:before_start_point, :after_start_point]
+    @derive {Jason.Encoder, only: [:before_start_point, :after_start_point]}
```

Comment:
question(non-blocking): why specify `only` here if it includes all keys?

Reply:
Given I wrote the first one with `@enforce_keys`, I'm not really sure why I did that in retrospect, I think just because I'd never used `@derive` before I was curious and it "seemed" right. I don't think I agree with my decision anymore, but it's not a big deal that it's like this.

(also, dang, I really wanted replying via email to work)